torch/csrc/jit/codegen/cuda/test/test_gpu_validator.h

```diff
@@ -452,6 +454,17 @@ inline void testValidate(
   }
 }
 
+inline void clearL2Cache() {
```

Comment:
Just copied from the benchmark directory as I'm doing ad-hoc perf testing in the C++ test files.